docs/sources/write/front-matter/index.md

```diff
@@ -202,9 +202,8 @@ Use `description` to provide the short description of the topic to search engine
 
 The number of characters vary by media, so make the description concise.
 Provide enough information to guide users to the content by describing what content the link leads to.
-Often, this doesn’t need to be original text, you can often scan the first few paragraphs to pluck the appropriate terms or phrases into the description.
-If it's too long, it is harmlessly truncated on social media.
-Use double quotes (`"`) to surround the title. Do not use smart quotes.
```

Comment:
This is not necessary and is potentially confusing because it affects the YAML parsing.